tests/org.jboss.tools.maven.ui.bot.test/src/org/jboss/tools/maven/ui/bot/test/apt/APTPropertiesPageTest.java

```diff
@@ -320,7 +323,7 @@ private void setProcessorCustomOutputFolder(String project, String customFolder)
 	
 	
 	
-	private void checkBuildPath(String project, List<String> paths, String additionalPath){
+	private void checkBuildPath(String project, List<String> paths, String additionalPath, boolean aptGeneratedTestsAdded, boolean generatedTestSourcesAdded){
```

Comment:
I would suggest different approach: instead of adding two booleans I would suggest to change method to checkBuildPath(String project, List<String> paths, List<String> additionalPaths), so in test you would be have something like:
List<String> additionalPaths = Arrays.asList(PROJECT_NAME+"/"+target+generatedSources+customFolder, ......);
checkBuildPath(PROJECT_NAME, defaultPaths, additionalPaths);

Reply:
Fixed, now testing on Jenkins.

Reply:
@jkopriva - Tested on Jenkins, passed:

<img width="1317" alt="passed on jenkins" src="https://user-images.githubusercontent.com/43820055/52653323-0dcbce80-2ef0-11e9-95b3-1cf7e819245d.png">